apps/np-atobarai/src/app/api/webhooks/saleor/fulfillment-tracking-number-updated/use-case.ts

```diff
@@ -30,21 +31,29 @@ type UseCaseExecuteResult = Promise<
   >
 >;
 
+export type SaleorOrderNoteServiceFactory = (authData: {
+  saleorApiUrl: SaleorApiUrl;
+  token: string;
+}) => IOrderNoteService;
```

Comment:
Maybe factory should receive a graphql client? I don't see a reason to go down with abstraction to build it again

Reply:
Fixed in https://github.com/saleor/apps/pull/2058/commits/b49bd66d21d9cbc9a87f863bdbb6cb91d2d05ac2